### PR TITLE
fix `Regression of the outerHTML property refactoring` (closes #1639)

### LINKED ISSUE #1639: Regression of the outerHTML property refactoring
In the line below the `this` keyword equals to element which `outerHTML` property is setting instead of the WindowSandbox instance.
https://github.com/DevExpress/testcafe-hammerhead/blob/44639ebbc4eda08a750581aa4419ebdabb18b5a4/src/client/sandbox/node/window.js#L1081

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

File: test/client/fixtures/sandbox/shadow-ui-test.js [hunk 1]
```diff
@@ -744,21 +745,39 @@ test('SVG elements\' className is of the SVGAnimatedString type instead of strin
...
 
             strictEqual(root.parentNode.parentNode.parentNode, iframe.contentDocument);
         });
 });
 
+test('setter shouldn\'t throw an error after rewriting body via iframe.body.outerHTML (GH-1639)', function () {
+    return createTestIframe()
+        .then(function (iframe) {
+            return new Promise(function (resolve, reject) {
+                iframe.contentWindow.onerror = function (e) {
```
Comment (churkin) [inline test/client/fixtures/sandbox/shadow-ui-test.js]:
```javascript
iframe.contentWindow.onerror = reject; 
```
:)

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@churkin FRP please

**Pull request merged**